site/index.html

```diff
@@ -167,6 +167,7 @@ <h2 id="downloads">Download and install</h2>
       I&apos;ll say that it&apos;s linked statically with <a href="https://www.musl-libc.org/">musl</a>, so it will work on any Linux-based OS,
       not just GNU/Linux.</span></dt>
   <dd><soupault-release platform="linux-x86_64" />
+  <dd>Nix: <a href="https://search.nixos.org/packages?channel=unstable&show=soupault&type=packages&query=soupault">nixpkgs</a></dd>
```

Comment:
The only thing I don't like is how that link is going to be in the middle of the links to binary builds.

Maybe add two separate sections under "Download and install": "Official builds" and "Distro packages"?

Reply:
This better?